src/components/MovieTile/__tests__/movieTitle.test.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+
+import MovieTitle from '../MovieTile';
+import { MovieType } from './../../../store/movies/types';
+
+describe('MovieDetails', () => {
+	const handleMovieClicked = jest.fn();
+
+	const movie: MovieType = {
+		imageUrl: 'https://via.placeholder.com/300x450.png?text=Movie+1',
+		movieName: 'Movie 1',
+		releaseYear: 2022,
+		genresList: ['Action', 'Horror'],
+		onMovieClick: handleMovieClicked,
+	};
+
+	it('renders MovieTitle with the values equal to passed in props', () => {
+		const { getByTestId } = render(<MovieTitle {...movie} />);
+		expect(getByTestId('movie-title-img').getAttribute('src')).toBe(
+			movie.imageUrl
+		);
+		expect(getByTestId('movie-title-movie-name')).toHaveTextContent(
+			movie.movieName
+		);
+		expect(getByTestId('movie-title-release-year')).toHaveTextContent(
+			movie.releaseYear.toString()
+		);
+		expect(getByTestId('movie-title-genres')).toHaveTextContent(
+			movie.genresList.join(', ')
+		);
+	});
+
```

Comment:
1. your test must be atomic (one test for one case)
2. if you create separate components for all subcomponents and create tests for them, you don't need to create test that check behaviour of each sub-component here.